reproductions/basic-sqlite/package.json

```diff
@@ -17,7 +18,7 @@
   },
   "devDependencies": {
     "@types/node": "17.0.45",
-    "prisma": "../../packages/cli",
+    "prisma": "^5.2.0",
```

Comment:
We would like to keep the relative import as that tests against the most up to date code.